src/DurableTask.Netherite.AzureFunctions/NetheriteProvider.cs

```diff
@@ -130,6 +133,25 @@ public override bool TryGetScaleMonitor(
             }
         }
 
+        public override bool TryGetTargetScaler(
+            string functionId,
+            string functionName,
+            string hubName,
+            string connectionName,
+            out ITargetScaler targetScaler)
+        {           
+            // Target Scaler is created per function id. And they share the same NetheriteMetricsProvider.
+            if ( this.metricsProvider == null)
+            {
+                this.loadPublisher ??= this.Service.GetLoadPublisher();
```

Comment:
Make loadPublisher class property too like metricsProvider so that it can be shared among scalers.